src/main/java/in/ashwanthkumar/gocd/slack/GoNotificationPlugin.java

```diff
@@ -18,6 +18,7 @@
 
 @Extension
 public class GoNotificationPlugin implements GoPlugin {
+    public static final String CONFIG_VIEW_TEMPLATE = "<div class=\"form_item_block\"><label>Message:</label></div>";
```

Comment:
Extract the HTML into a separate file in the resources. 

Ref
- https://github.com/ind9/gocd-s3-artifacts/blob/master/publish/src/main/resources/views/task.template.html
- https://github.com/ind9/gocd-s3-artifacts/blob/aabcdd7be5fa7fbba0d93dbb6574769d46edbaf5/publish/src/main/java/com/indix/gocd/s3publish/PublishTask.java#L54